appinventor/lib/blockly/src/core/block.js

```diff
@@ -826,6 +832,20 @@ Blockly.Block.prototype.showContextMenu_ = function(e) {
       options.push(disableOption);
     }
 
+    // Option to copy to backpack.
```

Comment:
We really want to minimize changes to blockly core. This can be done in the customMenuContext function in blocklyeditor.js (which is our code).